bridge-adaptivity/module/models.py

```diff
@@ -66,10 +68,16 @@ class Activity(models.Model):
     """
     General entity which represents problem/text/video material.
     """
+    LEVELS = (
+        ('low', _('low')),
+        ('medium', _('medium')),
+        ('high', _('high')),
+    )
+
     name = models.CharField(max_length=255)
     collection = models.ForeignKey('Collection', null=True)
-    tag = fields.CharField(max_length=255, blank=True, null=True)
-    difficulty = models.FloatField(verbose_name="max_points", blank=True, null=True)
+    tags = fields.CharField(max_length=255, blank=True, null=True)
+    difficulty = fields.CharField(choices=LEVELS, default='medium', max_length=32)
```

Comment:
I would use 'l', 'm', 'h' keys for LEVEL and decrease difficulty field's max-length to 1 character